packages/eslint/src/generators/convert-to-flat-config/__snapshots__/generator.spec.ts.snap

```diff
@@ -410,3 +450,30 @@ module.exports = [
 ];
 "
 `;
+
+exports[`convert-to-flat-config generator should handle custom eslintignores 1`] = `
+"const baseConfig = require('../../eslint.config.js');
```

Comment:
We'd previously discussed improving the newline spacing here in the converted output